exec/java-exec/src/main/java/org/apache/drill/exec/server/DrillbitContext.java

```diff
@@ -189,6 +192,10 @@ public ScanResult getClasspathScan() {
 
   public RemoteFunctionRegistry getRemoteFunctionRegistry() { return functionRegistry.getRemoteFunctionRegistry(); }
 
+  public DrillOperatorTable getOperatorTable() {
```

Comment:
Please add java doc explaining reasons why we are caching operator table and pointing out to the USE_DYNAMIC_UDFS_KEY which triggers cached operator table usage.